### ISSUE TITLE
[FEATURE] Dokumentenliste neu laden

### ISSUE THREAD

**Issue by LazyGuy21No** [jlawyerorg/j-lawyer-org#1870]
Wenn mehrere Personen gleichzeitig in einer Akte arbeiten, fehlt etwas die Übersicht. Es hilft aktuell wohl nur: Akte verlassen und neu laden.

Schön wäre ein automatisches Reload der Dokumentenliste (einstellbar, vielleicht mindestens 1x/Minute, Abstände können manuell verlängert werden). 

Alternativ (evtl. auch übergangsweise): manueller Reload.

**Comment by j-dimension** [comment]
Verweise auf #541

**Comment by stm9x9** [comment]
Ich glaube es geht nicht (nur) um das gleichzeitige Arbeiten an einem Dokument - bzw deren Verhinderung. Sondern daß man sieht, wenn neue Dokumente von anderen erstellt wurden oder ein Dokument umbenannt wurde etc. ohne dafür die Akte verlassen zu müssen (oder falls das reicht, den Dokumentenreiter verlassen)

**Comment by j-dimension** [comment]
> Ich glaube es geht nicht (nur) um das gleichzeitige Arbeiten an einem Dokument - bzw deren Verhinderung. Sondern daß man sieht, wenn neue Dokumente von anderen erstellt wurden oder ein Dokument umbenannt wurde etc. ohne dafür die Akte verlassen zu müssen (oder falls das reicht, den Dokumentenreiter verlassen).

Jo, so habe ich das auch verstanden und das Ticket daher nicht als Duplikat geschlossen, sondern offen gelassen. Es sind zwei verwandte, aber unterschiedliche Aspekte.
